xml/System.Threading/Timer.xml

```diff
@@ -244,8 +244,8 @@ Sub TimerCallback(state As Object)
       <Docs>
         <param name="callback">A <see cref="T:System.Threading.TimerCallback" /> delegate representing a method to be executed.</param>
         <param name="state">An object containing information to be used by the callback method, or <see langword="null" />.</param>
-        <param name="dueTime">The amount of time to delay before <paramref name="callback" /> is invoked, in milliseconds. Specify <see cref="F:System.Threading.Timeout.Infinite" /> to prevent the timer from starting. Specify zero (0) to start the timer immediately.</param>
-        <param name="period">The time interval between invocations of <paramref name="callback" />, in milliseconds. Specify <see cref="F:System.Threading.Timeout.Infinite" /> to disable periodic signaling.</param>
+        <param name="dueTime">The amount of time to delay before <paramref name="callback" /> is invoked, in milliseconds. Specify negative one (-1) milliseconds to prevent the timer from starting. Specify zero (0) to start the timer immediately. Specify zero (0) to start the timer immediately.</param>
```

Comment:
I thought you were going to change it to Timeout.InfiniteTimeSpan. This diff suggests you're changing it back. I'm confused

Reply:
It was changed on the int and long versions to use numbers and on the TimeSpan overloads to use the TimeSpan.Zero and Timeout.Infinite?

Unless I did something wrong I will check in a bit unless comment resolves.

Reply:
I had a few typos sorry.

Reply:
@danmosemsft, @stephentoub  please let me know if there are any further concerns.

Reply:
> It was changed on the int and long versions to use numbers and on the TimeSpan overloads to use the TimeSpan.Zero and Timeout.Infinite?

Why?

Timeout.Infinite is an integer.  Timeout.InfiniteTimeSpan is a TimeSpan.